### PR TITLE
Added login, register, and forgot password views.

### PR DESCRIPTION
**Pull request by cameron-dahl**
I have added login, register, and forgot password views to the project which is in the accounts app. For some reason, I couldn't use the reverse function for the success_url in the URL patterns but that should be fine. Also, I added an include in the main URLs file so I wouldn't have to import all the URL paths. This is my first contribution to anything so I hope I did everything right 😃

### CODE REVIEW

**Comment by ykdojo** [comment]
Thanks! Actually, I'm thinking of limiting the initial signup process to signing up with Twitter only - so maybe I'll integrate this code once we're ready to have more than that! (sorry)

**Comment by Camnooten** [comment]
No problem! That's okay, no hard feelings. Thanks for looking it over :) Should I close this PR for now?

**Comment by Camnooten** [comment]
Also as a suggestion, you could use something like [django-allauth](https://github.com/pennersr/django-allauth) for Twitter integration.

**Comment by ykdojo** [comment]
Hmm I would say let's keep it open for now for reference?

And thanks for the suggestion!

Actually, I was looking at [this tutorial](https://simpleisbetterthancomplex.com/tutorial/2016/10/24/how-to-add-social-login-to-django.html), and it suggests using social-auth-app-django. Any idea which one might be a better choice?

**Comment by ykdojo** [comment]
Okay sounds good! Maybe I'll try following that tutorial (or another one) for social-auth-app-django and see how it goes. If it's too hard, then I'll look into django-allauth :)

**Comment by Camnooten** [comment]
Aright sounds good!

File: accounts/views.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth import login, authenticate
+from django.contrib.auth.forms import UserCreationForm
+
+
```
Comment (Tipuch) [inline accounts/views.py]: What do you think about adding this ?
```suggestion
from django.views.decorators.http import require_http_methods

@require_http_methods(["GET", "POST"])
```

**Comment by ykdojo** [comment]
Closing this issue for now.

**Pull request closed without merging**